pages/reference-sdk-api-kit/proposetransaction.mdx

```diff
@@ -0,0 +1,342 @@
+import { Tabs } from 'nextra/components'
+
+# `proposeTransaction`
+
+Creates a new multi-signature transaction with its confirmations and stores it in the Safe Transaction Service.
+
+## Usage
+
+{/* <!-- vale off --> */}
+
+<Tabs items={['example.ts', 'setup.ts']}>
+  <Tabs.Tab>
+    ```typescript
+    import { ProposeTransactionProps } from '@safe-global/api-kit'
+    import { OperationType, SafeTransactionData } from '@safe-global/safe-core-sdk-types'
+    import { apiKit } from './setup.ts'
+
+    const safeTransactionData: SafeTransactionData = {
+      operation: OperationType.Call,
+      safeTxGas: '0x...',
+      baseGas: '123',
+      gasPrice: '123',
+      gasToken: '0x...',
+      refundReceiver: '0x...',
+      nonce: '10'
+    }
+
+    const transactionConfig: ProposeTransactionProps = {
+      safeAddress: '0x...',
+      safeTxHash: '0x...',
+      safeTransactionData,
+      senderAddress: '0x...',
+      senderSignature: '0x...',
+      origin: 'App name' // Optional
+    }
+
+    await apiKit.proposeTransaction(transactionConfig)
+    ```
+  </Tabs.Tab>
+  <Tabs.Tab>
+    ```typescript
+    import SafeApiKit from '@safe-global/api-kit'
+
+    export const apiKit = new SafeApiKit({
+      chainId: 1n, // Mainnet
+    })
+    ```
+  </Tabs.Tab>
+</Tabs>
+
+{/* <!-- vale on --> */}
+
+## Parameters
+
+### `safeAddress`
+
+- **Type:** `string`
+
+The Safe address.
+
+```typescript focus=2
+await apiKit.proposeTransaction({
+  safeAddress: '0x...',
+  safeTxHash: '0x...',
+  safeTransactionData: {
+    operation: OperationType.Call,
+    safeTxGas: '0x...',
+    baseGas: '123',
+    gasPrice: '123',
+    gasToken: '0x...',
+    refundReceiver: '0x...',
+    nonce: '10'
+  },
+  senderAddress: '0x...',
+  senderSignature: '0x...'
+})
+```
+
+### `safeTxHash`
+
+- **Type:** `string`
+
+The Safe transaction hash of the proposed transaction.
+
+```typescript focus=3
+await apiKit.proposeTransaction({
+  safeAddress: '0x...',
+  safeTxHash: '0x...',
+  safeTransactionData: {
+    operation: OperationType.Call,
+    safeTxGas: '0x...',
+    baseGas: '123',
+    gasPrice: '123',
+    gasToken: '0x...',
+    refundReceiver: '0x...',
+    nonce: '10'
+  },
+  senderAddress: '0x...',
+  senderSignature: '0x...'
+})
+```
+
+### `operation`
```

Comment:
This too should indicate that operation and the following parameters are part of the transaction data.